packages/view/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@axonivy/process-editor-view",
+  "version": "13.1.0-next",
+  "private": false,
+  "license": "(EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0)",
+  "author": "Axon Ivy AG",
+  "homepage": "https://developer.axonivy.com/",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/axonivy/process-editor-client"
+  },
+  "files": [
+    "lib",
+    "src"
+  ],
+  "dependencies": {
+    "@axonivy/ui-components": "~13.1.0-next.451",
+    "@axonivy/ui-icons": "~13.1.0-next.451"
+  },
+  "peerDependencies": {
+    "react": "^18.2 || ^19.0"
+  },
+  "devDependencies": {
+    "@types/node": "^22.10.7",
+    "@types/react": "^19.0.8",
+    "deepmerge-ts": "^5.1.0",
+    "jsdom": "^26.0.0",
```

Comment:
no tests do exist in this package, so this shouldn't be needed

Reply:
True, I'll remove it.